RHEL ansible versioning

dozdowski opened the pull request:
Modify the required version of ansible from 2.3.2 to 2.3.1 .  Initial testing of 2.3.1 is successful, and this is the version of ansible current to RHEL7 and RHEL6.

dozdowski committed:
RHEL ansible versioning

Modify the required version of ansible from 2.3.2 to 2.3.1 .  Initial testing of 2.3.1 is successful, and this is the version of ansible current to RHEL7 and RHEL6.

chambridge commented:
@dozdowski Can you also update https://github.com/quipucords/rho/blob/master/requirements.txt#L1

dozdowski committed:
RHEL ansible versioning

Modify the required version of ansible from 2.3.2 to 2.3.1 . Initial testing of 2.3.1 is successful, and this is the version of ansible current to RHEL7 and RHEL6.

dozdowski commented:
Updated!  Oops, thanks for the catch.
